### ISSUE TITLE
bug(skore): `CoefficientsDisplay` does not store label information in multiclass classification for `EstimatorReports` and `CrossValidationReports`

### ISSUE THREAD

**Issue by GaetandeCast** [probabl-ai/skore#2415]
### Describe the bug

Title. The label column in the internal dataframe contains only `NaNs` which means that `frame` and `plot` don't behave as they should (hue on the label or subplot by label available and no `label` column).

This does not affects the `ComparisonReport` for some reason.

The issue is likely due to the constructor `_compute_data_for_display` since the internal dataframe is wrong.

### Steps/Code to Reproduce

```python=
from sklearn.datasets import make_classification
from skore import train_test_split
from sklearn.linear_model import LinearRegression
from skore import EstimatorReport

X, y = make_classification(
    n_classes=3,
    n_features=5,
    n_clusters_per_class=1,
    random_state=0,
)

split_data = train_test_split(X, y, test_size=0.2, random_state=0, as_dict=True)
report = EstimatorReport(LinearRegression(), **split_data)

display = report.inspection.coefficients()
print(display.coefficients)
display.frame()
display.plot()
```

### Expected Behavior

- the `label` column in the internal dataframe contains values in {0,1,2} instead of only `NaN` value
- the `.frame()` output has a `label` column containing values in {0,1,2}
-  the plot shows three bars for each feature, with a hue defined by the class label

### Actual Behavior

<img width="958" height="822" alt="Image" src="https://github.com/user-attachments/assets/47207213-7ae8-4362-9f21-1260a9990d83" />

### Environment

```shell
main
```

**Comment by GaetandeCast** [comment]
I used a `LinearRegression` for a classification problem, the bug is me :)
The fact that no warning was raised is maybe concerning but it would be a scikit-learn problem.
